server/src/main/kotlin/com/kamelia/jellyfish/util/Extensions.kt

```diff
@@ -6,18 +6,38 @@ import com.kamelia.jellyfish.core.ExpiredOrInvalidTokenException
 import com.kamelia.jellyfish.core.InvalidUUIDException
 import com.kamelia.jellyfish.core.MissingHeaderException
 import com.kamelia.jellyfish.core.MissingParameterException
+import com.kamelia.jellyfish.core.MultipartParseException
+import com.kamelia.jellyfish.rest.core.pageable.PageDefinitionDTO
 import com.kamelia.jellyfish.rest.user.UserRole
+import io.ktor.http.ContentDisposition
+import io.ktor.http.HttpHeaders
 import io.ktor.http.content.PartData
 import io.ktor.http.content.forEachPart
 import io.ktor.server.application.ApplicationCall
 import io.ktor.server.application.call
 import io.ktor.server.auth.jwt.JWTPrincipal
 import io.ktor.server.auth.principal
+import io.ktor.server.request.contentType
+import io.ktor.server.request.receive
 import io.ktor.server.request.receiveMultipart
+import io.ktor.server.response.header
+import io.ktor.server.response.respondFile
 import io.ktor.util.pipeline.PipelineContext
+import java.io.File
 import java.util.UUID
 import org.jetbrains.exposed.dao.UUIDEntity
 
+suspend fun ApplicationCall.respondFile(file: File, name: String, type: String) {
+    response.header(
+        HttpHeaders.ContentDisposition,
+        ContentDisposition.Attachment
+            .withParameter(ContentDisposition.Parameters.FileName, name)
+            .toString()
+    )
+    response.header("Mime-Type", type)
```

Comment:
Shouldn't the header be `Content-Type` instead?

Reply:
As said in the commit message, this is semantically a different thing. The Content-Type is always an octet-stream when sending a file that is supposed to be downloaded (it is to be interpreted as literal bytebuffer, without meaning), therefore, if we do still want to keep that information somewhere, we need another header.
Ktor call.sendFile automatically sets the Content-Type anyway.